massa-async-pool/src/lib.rs

```diff
@@ -16,33 +16,32 @@
 //!
 //! ```json
 //! {
-//!     "sender": "xxxx",  // address that sent the message and spent max_gas*gas_price+coins on emission
+//!     "sender": "xxxx",  // address that sent the message and spent max_gas+coins on emission
```

Comment:
`max_gas` is not an amount of coins, but a quantity of gas.

=> address that sent the message and spent `fee+coins` on emission